asset-transfer-basic/application-typescript-hsm/package.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "asset-transfer-basic",
+    "version": "1.0.0",
+    "description": "Asset Transfer Basic contract implemented in TypeScript",
+    "main": "dist/index.js",
+    "typings": "dist/index.d.ts",
+    "engines": {
+        "node": ">=12",
+        "npm": ">=5"
+    },
+    "scripts": {
+        "lint": "tslint -c tslint.json 'src/**/*.ts'",
+        "pretest": "npm run lint",
+        "start": "npm run build && node dist/app.js",
```

Comment:
on a production note, I would suggest running the build and then packaging just the built code.